packages/components/src/core/InputDropdown/style.ts

```diff
@@ -25,12 +25,14 @@ const doNotForwardProps = [
   "sdsStyle",
 ];
 
+type IntentType = "negative" | "notice" | "positive";
```

Comment:
just curious, should "default" be part of the IntentType?

Reply:
Hey @tihuan ,

Thanks for the review! Since our theme doesn’t include a `"default"` color intent, I had to keep them separate. However, the `intent` prop includes `"default" | IntentType` to cover all possible values. 🫠

Reply:
OH!! That's why 😆 TIL thank you!